src/icons/fileIcons.ts

```diff
@@ -804,6 +804,7 @@ export const fileIcons: FileIcons = {
         { name: 'nest', fileNames: ['nest-cli.json', '.nest-cli.json', 'nestconfig.json', '.nestconfig.json'] },
         { name: 'moonscript', fileExtensions: ['moon'] },
         { name: 'percy', fileNames: ['.percy.yml'] },
+        { name: 'nasal', fileNames: ['nas'] },
```

Comment:
Do you mean `fileExtensions` instead of `fileNames`?

A file name is really fixed to files with the exact same name e.g. "nas" and not more. So if you want to have an icon for "sample.nas" you have to use `fileExtensions: ['nas']`.

```suggestion
        { name: 'nasal', fileExtensions: ['nas'] },
```

Reply:
the extension is sample.nas, so yes sorry I meant fileExtensions